content/en/blog/_posts/2025-04-15-wi-for-image-pulls.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: blog
+title: "From Secrets to Service Accounts: Kubernetes Image Pulls Evolved"
+date: 2025-04-15
+slug: wi-for-image-pulls
+author: >
+  [Anish Ramasekar](https://github.com/aramase) (Microsoft)
+---
+
+Kubernetes has steadily evolved to reduce reliance on long-lived credentials
+stored in the API.
+A prime example of this shift is the transition of Kubernetes Service Account (KSA) tokens
+from long-lived, static tokens to ephemeral, automatically rotated tokens
+with OpenID Connect (OIDC)-compliant semantics.
+This advancement enables workloads to securely authenticate with external services
+without needing persistent secrets.
+
+However, one major gap remains: **image pull authentication**.
+Today, Kubernetes clusters rely on image pull secrets stored in the API,
+which are long-lived and difficult to rotate,
+or on node-level kubelet credential providers,
+which allow any pod running on a node to access the same credentials.
+This presents security and operational challenges.
+
+To address this, Kubernetes is introducing **Service Account Token Integration
+for kubelet Credential Providers**, now available in **alpha**.
+This enhancement allows credential providers to authenticate with registries
```

Comment:
I would specify to Kubelet CPs here
```suggestion
This enhancement allows Kubelet credential providers to authenticate with registries
```

The credential providers don't authenticate with registries directly, maybe `...allows credential providers to request workload identity so that this can later be used for image pull authentication with the registry` ?